docs/wallets/decrediton/decrediton-setup.md

```diff
@@ -130,3 +130,15 @@ Press **Create Wallet** and your wallet setup is complete. You will be taken bac
 1. Select **Create Wallet**. Wait for the blockchain to be downloaded.
 
 1. The blockchain will now be scanned for any transactions belonging to this wallet.
+
+## Restore Existing Wallet from file
+
+Decred wallet file locations are:
+
```

Comment:
* Windows: `C:\Users\{Username}\AppData\Local\Decrediton\wallets\mainnet\{walletname}`
* macOS: `~/Library/Application Support/decrediton/wallets/mainnet/{walletname}`
* Linux: `~/.config/decrediton/wallets/mainnet/{walletname}`

Reply:
In addition to making these bullet points, I suggest using `%USERNAME%` instead of `{Username}` for Windows as that is a common environment variable that any windows user can copy and paste into the path bar.

Also I believe it is more common to use <> to clarify that is where a user needs to provide their own text, so you may want to use <wallet name>